packages/extensions/core/src/lib/pipeline/pipeline.ts

```diff
@@ -359,11 +359,12 @@ export async function runPipeline(configView: AutorestContext, fileSystem: IFile
 
     // you can have --pass-thru:FOO on the command line
     // or add pass-thru: true in a pipline configuration step.
+    const configEntry = context.GetEntry(node.configScope.last.toString());
     const passthru =
-      context.GetEntry(node.configScope.last.toString())["pass-thru"] === true ||
+      (configEntry && configEntry["pass-thru"] === true) ||
```

Comment:
```suggestion
      configEntry?.["pass-thru"] === true ||
```

you can use the `safe navigation operator` here instead.